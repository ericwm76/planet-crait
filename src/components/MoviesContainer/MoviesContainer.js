import React from 'react';
import './MoviesContainer.scss';
import initialMovies from '../../initialData';
import Movie from '../Movie/Movie'
import SideBar from '../SideBar/SideBar';

const MoviesContainer = ({selectMovie, name, quote, rank}) => {
  const movieCards = initialMovies.map(movie => {
      return (<Movie 
        key={movie.episode_id} 
        id={movie.episode_id} 
        title={movie.title} 
        year={movie.release_date}
        url={movie.url}
        img={movie.img}
        selectMovie={selectMovie}
        />
      )
    })  

  return (
    <section className='movies-container'>
      <SideBar name={name}
               quote={quote}
               rank={rank}/>
      {movieCards}
    </section>  
  )
}

export default MoviesContainer