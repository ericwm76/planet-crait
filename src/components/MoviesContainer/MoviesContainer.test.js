import React from 'react';
import { shallow } from 'enzyme';
import MoviesContainer from './MoviesContainer';

describe('MoviesContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MoviesContainer />)
  })

  it('should match snapshot with all data passing in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})