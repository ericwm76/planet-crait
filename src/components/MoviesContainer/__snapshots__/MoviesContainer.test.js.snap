// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesContainer should match snapshot with all data passing in correctly 1`] = `undefined`;
